bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1497,34 +1508,15 @@ private boolean approvalNeeded() {
         return approval != null && approval.needsApproval(pr.targetRef());
     }
 
-    private void updateApprovalNeededComment(Map<String, Boolean> additionalProgresses) {
+    private void postApprovalNeededComment(Map<String, Boolean> additionalProgresses) {
         var existing = findComment(APPROVAL_NEEDED_MARKER);
-        var existingUnapproved = false;
-        for (var entry : additionalProgresses.entrySet()) {
-            if (entry.getKey().endsWith("needs maintainer approval") && !entry.getValue()) {
-                existingUnapproved = true;
-                break;
-            }
-        }
-        StringBuilder messageBuilder = new StringBuilder();
-        if (existingUnapproved) {
-            messageBuilder.append("⚠️  @").append(pr.author().username())
-                    .append(" There are still some issues that have not received maintainer approval.")
-                    .append("\n")
-                    .append("Please follow the instruction here to get maintainer approval: ")
-                    .append("[Requesting push approval for fixes](https://openjdk.org/projects/jdk-updates/approval.html)");
-        } else {
-            messageBuilder.append("@").append(pr.author().username()).append(" All the issues have already got the maintainer approval!");
-        }
-        messageBuilder.append(APPROVAL_NEEDED_MARKER);
-        String message = messageBuilder.toString();
         if (existing.isPresent()) {
-            if (!existing.get().body().equals(message)) {
-                pr.updateComment(existing.get().id(), message);
-            }
-        } else {
-            pr.addComment(message);
+            return;
         }
+        String message = "⚠️  @" + pr.author().username() +
+                " This change has now been reviewed and requires maintainer [approval](" + approval.documentLink() + ")." +
```

Comment:
I did some more thinking about how this should be formulated. I don't think we should mention the review state here either. 
```suggestion
                " This change is now ready for maintainer [approval](" + approval.documentLink() + ")." +
```

Reply:
```suggestion
                " This change is now ready for you to apply for maintainer [approval](" + approval.documentLink() + ")." +
```

Reply:
Did you see my last suggestion here? I think we need to be clear that the ball is in the hands of the PR owner.

Reply:
Sorry, I missed it. Will fix it soon.